### DISCUSSION TITLE
Help. I don't really understand how to use it..

### DISCUSSION THREAD

**Discussion by thunderwin** [uicrooks/shopify-theme-lab#129] [Q&A]
I can use VUE,   I know a little Shopify theme..

But, I don't really understand how to use this tool..

should I just write Vue component instead of liquid?  

or do you have a clear tutorial ?

**Comment by sergejcodes** [comment]
There's is no tutorial, but there's a [theme](https://github.com/uicrooks/shopify-foundation-theme) which is written with this Tool. It can provide good guidance. 

As a best practice, I would say you should write as much as possible with Shopify's liquid templating language and use Vue components only when you really need them. The Vue components aren't meant to replace the liquid templates but augment them when needed.
